tracker/tracker/src/webworker/index.ts

```diff
@@ -9,12 +9,19 @@ import {
 import QueueSender from "./QueueSender.js";
 import BatchWriter from "./BatchWriter.js";
 
-
+enum WorkerStatus {
+  NotActive,
+  Starting,
+  Stopping,
+  Active
+}
```

Comment:
What I meant by adding "Stopping" to the app `ActivityState`.  There are a lot of things happening between `this.worker.postMessage("stop")` which deletes the `writer` here, and  the actual `this.activityState = ActivityState.NotActive`. This causes that error in a worker once in a while.

Another one possible source of it:  the `this.activityState = ActivityState.Active` in the `_start()` function of `App` should be placed after the auth `postMessage`.

(Though  it is also good to have  a Worker status here for making codeflow understandable)